balance-management/balance-manager/wasmcloud.toml

```diff
@@ -0,0 +1,8 @@
+name = "balance_manager"
```

Comment:
Change this name to concrete name to be Balance Manager as it's the actor name appears in UI to be in good shape